src/c++/perf_analyzer/inference_profiler.cc

```diff
@@ -1474,6 +1479,22 @@ InferenceProfiler::SummarizeServerStats(
   return cb::Error::Success;
 }
 
+void
+InferenceProfiler::SummarizeOverhead(
+    const uint64_t window_duration_ns, const uint64_t idle_ns,
+    PerfStatus& summary)
+{
+  // It is possible for the idle to be larger than the duration, based on
+  // the way idle windows are added. For that case, just mark 0% overhead
```

Comment:
Just to make sure I'm understanding correctly, basically idle time is the amount of time that each thread is waiting for the server to response or enqueue the requests using the `client_backend_` APIs? This could be zero percent because the model is taking long time to respond? I don't understand why idle time has a relationship to window size. For example, if a model is taking a long time to respond and it has large inputs the "overhead" could be still zero in some windows?

Reply:
That comment is stale and has been removed. I am now using a class to track the idle time, which can handle the case that my first revision couldn't handle.